src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -677,12 +677,8 @@ MulticopterPositionControl::reset_pos_sp()
 	if (_reset_pos_sp) {
 		_reset_pos_sp = false;
 		/* shift position setpoint to make attitude setpoint continuous */
-		_pos_sp(0) = _pos(0) + (_vel(0) - PX4_R(_att_sp.R_body, 0, 2) * _att_sp.thrust / _params.vel_p(0)
```

Comment:
Why did this get removed? Did you carefully check the rate outputs before and after?